### DISCUSSION TITLE
AO Cache directory?

### DISCUSSION THREAD

**Discussion by rayw45** [kubilus1/autoortho#416] [General]
Two questions.

Firstly is it truly okay to keep using my now very large 400GB cache first used with v 4.4 with the new latest updates to AO (0.7.0) as I would rather not have all the pauses with new tiles being redownloaded as I fly when I already have them in many areas I regularly fly.

Secondly. With SSD hard drives now so cheap can the Cache folder size be increased from 500Gb to say 1TB as I think many here do not mind even dedicating a separate 1TB SSD drive for the cache. So what happens now when I reach a cache size of 500GB?  Does it delete the oldest cached tile files and save the new ones or does it just fail to store any more tiles??

**Comment by Jeff2122** [comment]
@kubilus1 would have an answer, but I have an opinion: 
First, the scenery is fundamentally the same for all versions so the 400GB cache data should be just fine to use. Only the z_autoortho data is arranged differently in versions prior to v0.5.0. 
Second, the cache size can be whatever you can afford. I have never tested one larger than 50GB though. That size seems to work great for my needs. 
I haven't filled my cache up because I keep deleting it for test purposes, so I can't vouch for how it handles the files when it gets full. Does it trim to leave room for new files, or what? I can only guess that it will keep adding files over your 400GB limit and they will stay there until you press the clean button. But how big can it get?

**Reply by kubilus1** [reply]
Yeah, that's all pretty much correct.  You can have the cache be as big as you want.  The slider just went to 500GB because that seemed 'big enough' it won't do anything until you hit 'clean'.  I'm not entirely certain if there is some practical limit on the number of items in a single folder, though.  But if it's working now, it should be okay.

**Reply by Jeff2122** [reply]
Quick Google says Microsoft says:
"You can put 4,294,967,295 files into a single folder if drive is formatted with NTFS (would be unusual if it were not) as long as you do not exceed 256 terabytes (single file size and space) or all of disk space that was available whichever is less."

**Reply by Jeff2122** [reply]
That should be plenty for anyone, even @rayw45.

**Reply by rayw45** [reply]
Thanks guys for that input.  Yes I am quite happy to have even a complete dedicate 1TB SSD drive to get cached ortho tiles which avoid the mid flight pauses on routes flown before.  Not sure how large my cache folder is now, as when I check it Windows File Explorer takes over 30 minutes to count the number of files and cache folder size as it now has so many millions of tiles I now have cached.

My only remaining major problem left with AO is of course still this awful slowdown need to make debug = True for AO to even work with XP12 in Windows 10, using either Dokan or WinFSP.  So will wait for later AO versions for this problem to hopefully be fixed and put up with the slowdown for now as AO is so good it is worth that caveat.

**Reply by Jeff2122** [reply]
The only solution I've seen so far for one person who has the issue - remove overclock settings. If the overclock thing doesn't apply to you, we need to keep hunting for a solution.

**Reply by rayw45** [reply]
No Jeff that does not help for me.  The only overclocking I have is on the graphics card and that is carefully safely overclocked using stress test programs from MSI and 3DBench. So I switch my GFX card to default no OC and it makes no difference as I still need debug = True for AO 7 or 6 to get past the reading new scenery stage.   Never happened with 4.4 so I would have thought that the developers would be able to discover why this debug delay is now necessary and was not with 4.4 ??? 

Oh well at least AO 7 is worth putting up with the loading delay with the debugging process.

**Reply by rayw45** [reply]
Jeff as I see you are also on the X Plane org forum I am desperate for help as I cannot log into it all of a sudden either in Vivaldi or Chrome browsers.  Do they have a problem as I am trying everything to get in.  Vivaldi just displays a small black and white icon depicting a dead bird lying on its back, and Chrome just reports Aw Snap. I have cleared browsing data and cookies all to no avail. It shows I am logged in so not a password problem.  I can log into the store okay and my account there but not the forum.

**Reply by Jeff2122** [reply]
It seems to be working this morning from here. I catch up each morning, and today was no different. Sorry I can't be more helpful...
